generate.py

```diff
@@ -168,21 +156,15 @@ def get_gen_mot_np(args, generated_motion, mean_joints, std_joints):
     return generated_motion, index
 
 
-def generate(args, g_ema, device, mean_joints, std_joints, entity):
+def generate(args, g_ema: Generator, device, mean_joints: torch.tensor, std_joints: torch.tensor,
+             motion_statics: StaticData, entity: str):
 
     type2func = {'interp': interpolate, 'sample': sample, 'edit': edit}
     with torch.no_grad():
         g_ema.eval()
         mean_latent = g_ema.mean_latent(args.truncation_mean)
         generated_motions = type2func[args.type](args, g_ema, device, mean_latent)
 
-    if entity.str() == 'Joint':
```

Comment:
are you sure there is no need for a special care for Joint?

Reply:
Yes, this section only prepare the edge_rot_dict var differently wether we're using Joint or Edge entity, since we don't need that variable anymore, we don't need that if as well.
The real separation between Joint and Edge will arrive later on line 208 in the method motion2fig.